lib/reporters/console.js

```diff
@@ -13,6 +14,12 @@ const internals = {
     width: 50
 };
 
+internals.stringify = (obj, serializer, indent, decycler) => {
+
+    return StableStringify(
```

Comment:
if you do something like
```javascript
internals.stringify = (obj, serializer, indent) => {

    return StableStringify(
      JSON.parse(StringifySafe(obj)),
      { space: indent, replacer: serializer });
};
```

The test 'console reports with undefined values fails'

After the first stringify we lose some information about the original object:
```javascript
JSON.stringify({a: undefined}) === '{}'
JSON.stringify({a: Infinity}) === '{"a":null}'
```